### DISCUSSION TITLE
編集履歴の各履歴に直接ジャンプできるURLがほしい

### DISCUSSION THREAD

**Discussion by JunichiIto** [increments/qiita-discussions#924] [要望・アイデア・不具合]
https://qiita.com/jnchito/items/42193d066bd61c740612#comment-363a6b470278c5aabebf に以下のようなコメントを書きました。

![Screenshot 2024-11-11 at 19 00 11](https://github.com/user-attachments/assets/edc67388-a732-45f1-86bf-a3e11ea59ba2)

本当は2019年7月22日の「describe => RSpec.describe に修正しました。」という編集履歴を直接表示するURLを貼りたかったのですが、そのURLがないため、やむを得ず編集履歴トップのURLを貼りました。（今回が初めてではなく、同じように困ったことが過去に数回あります）
というわけで、「ここで変更してますよ」という履歴を相手に伝えるためのURL（アンカー付きURL？）があると嬉しいです。

あと、これは優先度低めですが、「describe => RSpec.describe に修正しました。」という文言を編集履歴のページからコピーしようと思ってドラッグアンドドロップしても、なかなかテキストを選択できなくて苦労しました。
この点も改善されると嬉しいです。

![Nov-11-2024 19-06-36](https://github.com/user-attachments/assets/952a7ae7-92ae-400e-aaee-bf21de22f9b1)

ご検討よろしくお願いします。

**Comment by tanaka-asami415** [comment]
@JunichiIto 
リクエストありがとうございます！

1. 編集履歴の各履歴に直接ジャンプできるURL
1. 編集履歴のテキストコピーがしにくいのを改善する

1については、あるととても便利そうです。2については、ご不便をおかけして申し訳ありません🙏
時期はお約束できませんが、両方とも改善したいと思います！

**Comment by tsuzuki-takaaki** [comment]
@JunichiIto 
お待たせしました！

記事の各編集履歴に固有のURLを付与するように変更し、履歴ごとの共有ができるようになりました。
https://qiita.com/release-notes/2025-06-10

ぜひご活用いただければと思います！

「編集履歴のテキストコピーがしにくいのを改善する」に関しましては、改善の検討を進めておりますので、お時間をいただけますと幸いです🙇‍♂️

**Reply by JunichiIto** [reply]
@tsuzuki-takaaki おお、どうもありがとうございます！助かります 😄

**Reply by yuta0709** [reply]
@JunichiIto 
お待たせいたしました！

編集履歴のテキストがコピーしやすいように改善いたしました。
ぜひお試しください！

今後ともQiitaをよろしくお願いいたします🙇

![recording](https://github.com/user-attachments/assets/e7dbcabc-2667-4317-8706-108b4ea7d85b)

**Reply by JunichiIto** [reply]
@yuta0709 改善されていることを確認しました。ご対応どうもありがとうございました！ 😄
